src/amo/components/LanguagePicker.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
+
+import translate from 'core/i18n/translate';
+
+import './LanguagePicker.scss';
+
+
+export class LanguagePickerBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { i18n } = this.props;
+
+    return (
+      <div className="LanguagePicker">
+        <h3 className="LanguagePicker-header">
+          {i18n.gettext('Browse in your language')}
+        </h3>
+        <select className="LanguagePicker-selector"
+          ref={(ref) => { this.selector = ref; }}>
+          <option value="en-US">English (US)</option>
+        </select>
+      </div>
+    );
+  }
+}
+
+export default translate({ withRef: true })(LanguagePickerBase);
```

Comment:
Hah, I was wondering if it would post these in the order I wrote them or the order of the files. It's the order you write them apparently. Later it says:

> Using compose here would make this a little easier to read.

> ```js
export default compose(
  translate({ withRef: true }),
)(LanguagePickerBase);